### ISSUE TITLE
JS/HTML injection via SSID

### ISSUE THREAD

**Issue by jn4kr** [samdenty/Wi-PWN#90]
As you can see it's possible to inject HTML/Javascript in the attack page via the ssid. 

I don't consider it as a future.

### Screenshots
![scan](https://user-images.githubusercontent.com/11200749/54484516-b4e1b400-4868-11e9-9695-67f446d220e2.png)

![attack](https://user-images.githubusercontent.com/11200749/54484515-b4e1b400-4868-11e9-9a93-1f3d3e6f58a6.png)

### Steps to reproduce

1. Create a beacon with the tag you wan't to inject as SSID (e.g. <svg onload=\"alert('1')\"/> )
2. Select the beacon as target
3. Switch to the attack page

**Comment by minanagehsalalma** [comment]
@jn4kr  ...
Does android even process these WiFi names in the available wifis list ?!

**Comment by jn4kr** [comment]
Yeah. But i think it's off topic, so I'll sent you a mail about this topic.

**Comment by StijnSimons** [comment]
Project is kinda dead but once i really don’t know what to do, i’ll think about it.

**Comment by kingdevnl** [comment]
We might be able to strip out all HTML code using a REGEX?

**Comment by Helixable** [comment]
Check thsi out: https://code-examples.net/en/q/129e949
